samples/nrf9160/fmfu_smp_svr/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _fmfu_smp_svr_sample:
+
+Full modem update over SMP Server Sample
+########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Full modem firmware update sample implements a Simple Management Protocol (SMP) server.
+SMP is a basic transfer encoding for use with the MCUmgr management protocol.
+For more information about MCUmgr and SMP, please see :ref:`device_mgmt`.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160ns
+
+.. include:: /includes/spm.txt
+
+Overview
+********
+
+The sample first deinitializes the :ref:`nrfxlib:nrf_modem`.
+Next, it registers a stat command and firmware upload and get hash commands to
+MCUMgr.
+
+The sample then goes into and idle loop waiting for communication over the serial line.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/fmfu_smp_svr`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+
+Testing
+=======
+
+After programming the sample to your board, test it by performing the following steps:
+
+1. Connect the USB cable and power on or reset your nRF9160 DK.
+#. Open a terminal emulator and observe that the sample starts, close the terminal emulator. 
```

Comment:
Is there a print which indicates that the sample is ready to receive the data?

Reply:
yes